kubert/src/admin.rs

```diff
@@ -238,3 +250,22 @@ fn handle_ready(Readiness(ready): &Readiness, req: Request<Body>) -> Response<Bo
             .unwrap(),
     }
 }
+
+fn handle_metrics(
+    metrics: &PrometheusHandle,
+    process: &Collector,
+    req: Request<Body>,
+) -> Response<Body> {
+    process.collect();
+    match *req.method() {
+        hyper::Method::GET | hyper::Method::HEAD => Response::builder()
```

Comment:
it doesn't seem correct to handle `HEAD` requests by rendering all the metrics? shouldn't we just be sending the `Content-Length` and other headers with an empty body when receiving a `HEAD` request?

Reply:
how do you know the content-length?

Reply:
you would have to render the body, but not send it on the wire --- my understanding of `HEAD` (and correct me if i'm wrong?) is that it's explicitly requesting to not receive a body.